src/@types/event.ts

```diff
@@ -368,3 +372,15 @@ export interface StateEvents {
     // MSC3672
     [M_BEACON_INFO.name]: MBeaconInfoEventContent;
 }
+
+export interface AccountDataEvents {
+    [EventType.PushRules]: IPushRules;
+    [EventType.Direct]: { [userId: string]: string[] };
```

Comment:
I feel like we should have separate types for room-level account data and global account data.

Reply:
m.direct is *not* global account data

Reply:
Except it literally is... https://spec.matrix.org/v1.12/client-server-api/#mdirect

Reply:
ugh, sorry, you're right.